install/upgrade_to_2.13.php

```diff
@@ -114,4 +114,14 @@ protected function fixInconsistency() {
       $this->migration->changeField($table, 'plugin_formcreator_formanswers_id', 'plugin_formcreator_formanswers_id', 'integer', ['value' => '0']);
       $this->migration->changeField($table, 'plugin_formcreator_questions_id', 'plugin_formcreator_questions_id', 'integer', ['value' => '0']);
    }
+
+   public function migrateCategory() {
+      $table = 'glpi_plugin_formcreator_forms';
+      $this->migration->addField($table, 'knowbaseitemcategories_id', 'string', ['value' => '', 'after' => 'plugin_formcreator_categories_id']);
+      $query = "UPDATE `$table` f
+               INNER JOIN `glpi_plugin_formcreator_categories` fc
+                  ON fc.`plugin_formcreator_categories_id` = f.`plugin_formcreator_categories_id`
+               SET f.`knowbaseitemcategories_id` = fc.`knowbaseitemcategories_id`;";
+      $this->migration->addPostQuery($query);
+   }
```

Comment:
You also need to drop the table glpi_plugin_formcreator_categories in the upgrade procedure.